lms/envs/aws.py

```diff
@@ -590,6 +590,8 @@
             'schedule': datetime.timedelta(hours=ENV_TOKENS.get('THIRD_PARTY_AUTH_SAML_FETCH_PERIOD_HOURS', 24)),
         }
 
+    RESPECT_X_FORWARDED_HEADERS = ENV_TOKENS.get('RESPECT_X_FORWARDED_HEADERS')
```

Comment:
Would be better to call this `SOCIAL_AUTH_RESPECT_X_FORWARDED_HEADERS` if it's only used by third party auth. This line (and the tests) is all you'd need to change, since the `self.setting()` method used to read this will try with and without the `SOCIAL_AUTH_` prefix.